src/components/index.ts

```diff
@@ -36,6 +36,10 @@ import RuiSkeletonLoader, { type Props as SkeletonLoaderProps } from '@/componen
 import RuiMenuSelect, { type Props as MenuSelectProps } from '@/components/forms/select/RuiMenuSelect.vue';
 import RuiAccordions, { type Props as AccordionsProps } from '@/components/accordions/accordions/RuiAccordions.vue';
 import RuiAccordion, { type AccordionProps } from '@/components/accordions/accordion/RuiAccordion.vue';
+import {
+  type BottomSheetProps,
+  default as RuiBottomSheet,
+} from '@/components/overlays/bottom-sheet/RuiBottomSheet.vue';
```

Comment:
can you change this to be consistent with the rest? wi
```suggestion
import RuiBottomSheet, { type BottomSheetProps } from '@/components/overlays/bottom-sheet/RuiBottomSheet.vue';
```